integ-test/src/test/java/org/opensearch/sql/legacy/OpenSearchSQLRestTestCase.java

```diff
@@ -252,15 +266,26 @@ protected static void configureHttpsClient(RestClientBuilder builder, Settings s
    * Initialize rest client to remote cluster,
    * and create a connection to it from the coordinating cluster.
    */
-  public void configureMultiClusters() throws IOException {
-    initRemoteClient();
+  public void configureMultiClusters(String remote)
+      throws IOException {
+    initRemoteClient(remote);
 
     Request connectionRequest = new Request("PUT", "_cluster/settings");
-    String connectionSetting = "{\"persistent\": {\"cluster\": {\"remote\": {\""
-        + REMOTE_CLUSTER
-        + "\": {\"seeds\": [\""
-        + getTestTransportCluster(REMOTE_CLUSTER).split(",")[0]
-        + "\"]}}}}}";
+    String connectionSetting = String.format(
```

Comment:
I would make this JSON a static string for the class

Reply:
Updated in a618be8f6